src/vpn-api/utils/haproxy.ts

```diff
@@ -0,0 +1,47 @@
+/*
+	Copyright (C) 2019 Balena Ltd.
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published
+	by the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import * as Promise from 'bluebird';
```

Comment:
I think most other places, we're switching to `import * as Bluebird from 'bluebird'`.

Reply:
Interesting, okay. Do you know why?

Reply:
I believe it's in a move to slowly move to native Promises where possible, only using Bluebird as sugar.

Reply:
Fair enough, I can certainly see the argument for using native Promises but Bluebird's sugar certainly does… _make the medicine go down_.

I'll get my coat.